101-hdinsight-hbase-replication-geo/azuredeploy.json

```diff
@@ -21,6 +21,44 @@
         "description": "Location for all resources."
       }
     },
+    "Vnet1Region": {
+      "type": "string",
+      "defaultValue": "West US",
```

Comment:
need to remove the defaultValue and allowedValues... and then supply param files for both MAC and FF to set appropriate locations...  (see contribution guide or just look for another sample that contains a file called azuredeploy.parameters.us.json